File: 10001_prime/python/10001_prime.py
primeNums = 0
latestPrime=0
current=2

def isPrime(a):
	for x in xrange(2,a):
		if a % x == 0:
			return False
	return True

while primeNums != 10001:
	if isPrime(current):
		primeNums+=1
		latestPrime = current
	current += 1

print latestPrime

File: 10001_prime/php/10001_prime.php
<?php

$primeNumsSoFar=0;
$latestPrime=0;
$current=2;

function checkPrime($a) {
	foreach (range(2,$a-1) as $x) {
		if ($a%$x==0) {
			return false;
		}
	}
	return true;
}

while ($primeNums != 10000) {
	if (checkPrime($current)) {
		$primeNums++;
		$latestPrime = $current;
		echo $latestPrime."\n";
	}
	$current++;
}

echo $latestPrime;
